src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -6322,6 +6280,226 @@ class C
             Assert.True(tuple1.Equals(tuple4, TypeCompareKind.IgnoreDynamicAndTupleNames));
         }
 
+        [Fact]
+        [WorkItem(36047, "https://github.com/dotnet/roslyn/issues/36047")]
+        public void CreateTupleTypeSymbol_UnderlyingType_DefaultArgs()
+        {
+            var source =
+@"class Program
+{
+    (int, string) F;
+}";
+            var comp = CreateCompilation(source);
+            var tuple1 = (INamedTypeSymbol)((IFieldSymbol)comp.GetMember("Program.F")).Type;
+            var underlyingType = tuple1.TupleUnderlyingType;
+
+            var tuple2 = comp.CreateTupleTypeSymbol(underlyingType);
+            Assert.True(tuple1.Equals(tuple2));
+
+            tuple2 = comp.CreateTupleTypeSymbol(underlyingType, default);
```

Comment:
>default [](start = 64, length = 7)

consider naming args in various calls

Reply:
Calls with named arguments are below.

---
In reply to: [302727657](https://github.com/dotnet/roslyn/pull/37048#discussion_r302727657) [](ancestors = 302727657)